pkg/swagger/typewalker.go

```diff
@@ -88,6 +88,7 @@ func (tw *typeWalker) schemaFromType(t types.Type, deps map[*types.Named]struct{
 			s.Type = "boolean"
 		case types.Int:
 			s.Type = "integer"
+			s.Format = "int64"
```

Comment:
Why this change? It does seem to affect specs for older versions. Is it expected?

Reply:
Removed. See original PR documented in the description